packages/dart_frog_cli/lib/src/dev_server_runner/dev_server_runner.dart

```diff
@@ -233,7 +233,7 @@ class DevServerRunner {
   ///
   /// This method will throw a [DartFrogDevServerException] if called after
   /// [stop] has been called.
```

Comment:
Nit:

```suggestion
  /// [stop] has been called.
  ///
  /// The [arguments] are optional and usually expected to be those from [ArgResults.rest]; since a `--` signals
  /// the **end of options** and disables further option processing from `dart_frog dev`. Any [arguments]
  /// after the `--` are passed into the [Dart tool](https://dart.dev/tools/dart-tool) process. 
  ///
  /// For example, you could use the `--` to enable [experimental flags](https://dart.dev/tools/experiment-flags)
  /// such as macros by doing `dart_frog dev -- --enable-experiment=macros`.
```